test/extended/testdata/marketplace/csc/01-csc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1
+kind: CatalogSourceConfig
```

Comment:
It would be great if we can replace these YAML fils with `Template` so that others can reuse it flexible in the future.